src/app.tsx

```diff
@@ -11,21 +11,28 @@ import { CatContextProvider } from './contexts';
 import { ErrorFallback } from './components';
 import { ThemeProvider } from '@rneui/themed';
 import appTheme from './app-theme';
+import { DatadogProvider } from '@datadog/mobile-react-native';
+import DatadogConfig from './services/datadog';
+import Logger from './logger/logger';
 
 const App = () => {
+  Logger.initializeLoggers();
   const ErrorComponent = useCallback(() => <ErrorFallback />, []);
+
   return (
     <ErrorBoundary
-      onError={(e, stack) => console.error(`App Error: ${e} ${stack}`)}
+      onError={(e, stack) => Logger.error(`App Error: ${e} ${stack}`)}
       FallbackComponent={ErrorComponent}
     >
-      <SafeAreaProvider initialMetrics={initialWindowMetrics}>
-        <CatContextProvider>
-          <ThemeProvider theme={appTheme}>
-            <ApplicationNavigator />
-          </ThemeProvider>
-        </CatContextProvider>
-      </SafeAreaProvider>
+      <DatadogProvider configuration={DatadogConfig}>
+        <SafeAreaProvider initialMetrics={initialWindowMetrics}>
```

Comment:
DatadogProvider is at lower level and we are using the Logger above it, will that work for error boundary?

Reply:
Thanks @codVenturer for the feedback. I tried to test this scenario and i was able to see the log errors in the Datadog log manager. Have a look at the screenshot. Pls let me know if you see any issues with your env and we can work together to resolve it.
![Uploading Screenshot 2023-12-26 at 12.17.33 PM.png…]()